server/services/notify/notifysubscriptions/diff.go

```diff
@@ -200,13 +195,22 @@ func (dg *diffGenerator) generateDiffsForCard(card *model.Block, schema model.Pr
 		}
 		cardDiff.Diffs = childDiffs
 	}
+	cardDiff.Authors.Append(authors)
+
 	return cardDiff, nil
 }
 
 func (dg *diffGenerator) generateDiffForBlock(newBlock *model.Block, schema model.PropSchema) (*Diff, error) {
+	dg.logger.Debug("generateDiffForBlock - new block",
+		mlog.String("block_id", newBlock.ID),
+		mlog.String("block_type", string(newBlock.Type)),
+		mlog.String("modified_by", newBlock.ModifiedBy),
+		mlog.Int64("update_at", newBlock.UpdateAt),
+	)
+
 	// find the version of the block as it was at the time of last notify.
 	opts := model.QueryBlockHistoryOptions{
-		BeforeUpdateAt: dg.lastNotifyAt,
+		BeforeUpdateAt: dg.lastNotifyAt + 1,
```

Comment:
Nit: this can be unprecise, and can lead to very strange bugs in very weird times (Not a big deal, but with enough bad luck, probably you can miss s change in the notifications.

I wouldn't fix it here, and I don't even know if I would fix that, but at least we should know it.

Reply:
This query is to find the version of the block that is older than or equal to the last time a notification was sent.  I'm not seeing how it could miss a change.  The +1 is simply to turn the `X < updateAt` to `X <= updateAt`.  Am I missing something?